apps/portals-base/src/style/_variables.scss

```diff
@@ -9,19 +9,19 @@ $header-height: 75px;
 $banner-height: 47px;
 $border-color-gray: #dcdcdc;
 
-// Since scss breaks if a file is missing in a local url link these functinons must get manually overwitten
-@function getHeader($path) {
+// Since scss breaks if a file is missing in a local url link these functions must get manually overwitten
+@function getHeader() {
```

Comment:
$path arg is no longer necessary, I believe it was only needed for dynamic portal configurations